x-pack/platform/packages/shared/kbn-ai-assistant/src/chat/knowledge_base_callout.tsx

```diff
@@ -1,122 +0,0 @@
-/*
```

Comment:
I couldn't find any usages of this.
@arturoliduena looks ok to delete to you?

Reply:
Looks like you removed it in https://github.com/elastic/kibana/pull/214133 as well :)